src/main/java/frc/robot/Robot.java

```diff
@@ -27,6 +27,7 @@
   public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
+    
```

Comment:
Usually we put the code directly under the comment that describes it, no blank line.